runtests

```diff
@@ -158,7 +163,8 @@ test_all() {
   fi
 }
 
-: LOGFILE=""
+# Default value
+: "${LOGFILE:="./lkvs.log"}"
```

Comment:
Hi Pengfei, is this correct ?

Reply:
As before I wrote as following:
: ${LOGFILE:="./lkvs.log"}
And met below warning:
https://www.shellcheck.net/wiki/SC2223

And fixed above warning to write as present sample.

The double quotes within the curly braces will not conflict with the double quotes outside the curly braces, and there is no problem with the test.

Thanks!